torchgeo/datasets/bigearthnet.py

```diff
@@ -495,7 +504,7 @@ def _download(self, url: str, filename: Path, md5: str) -> None:
             filename: output filename to write downloaded file
             md5: md5 of downloaded file
         """
-        if not os.path.exists(filename):
+        if not os.path.exists(os.path.join(self.root, filename)):
```

Comment:
Saw this, and actually I think this is required, because it should check whether the file exists in root already right?

Reply:
Good catch. Actually, this line isn't necessary because we already check if the zipfile exists and extract it before we reach the download. Can you open a separate PR to fix this so we can backport it to 0.6.3?